src/application-manager.js

```diff
@@ -28,7 +28,7 @@ import { TargetStateAccessor } from './device-state/target-state-cache';
 
 import { ServiceManager } from './compose/service-manager';
 import { Service } from './compose/service';
-import { Images } from './compose/images';
+import * as images from './compose/images';
```

Comment:
I'm in favour of making all imports of the image module to be `Images` not that it is a singleton. This first letter uppercase convention for singletons is something I've seen before and provides an indicator that it is a singleton rather then a variable that exists as multiple copies.

Doing that would also lets us resolve naming conflicts that had to be fixed like `images` being `imgs`:
https://github.com/balena-io/balena-supervisor/blob/abeb33ae1fd9b14e014c267df9c903368bbe064c/src/application-manager.js#L264

Examples like this are found throughout the PR so if implemented we should make sure to follow this convention.